controllers/attractor/nse-vlan.go

```diff
@@ -114,6 +120,11 @@ func (i *NseDeployment) insertParameters(dep *appsv1.Deployment) *appsv1.Deploym
 				container.Ports = append([]corev1.ContainerPort{}, corev1.ContainerPort{HostPort: common.VlanNsePort, ContainerPort: common.VlanNsePort})
 			}
 			container.Env = i.getEnvVars(container.Env)
+			// set resource requirements for container (if not found, then values from model
+			// are kept even upon updates, as getReconciledDesiredStatus() overwrites containers)
+			if err := common.SetContainerResourceRequirements(&i.attractor.ObjectMeta, &container); err != nil {
+				i.exec.LogInfo(fmt.Sprintf("attractor %s, %v", i.attractor.ObjectMeta.Name, err))
```

Comment:
LogError would be more appropriate

Reply:
Initially I was using LogError but that also printed a stack dump which I found an overkill and unnecessary spam, thus setlled with LogInfo